src/main/java/org/itech/fhirhose/etl/service/impl/ETLServiceImpl.java

```diff
@@ -221,6 +238,39 @@ private void putPractitionerValuesIntoETLRecord(ETLRecord etlRecord, Practitione
 //					+ fhirPractitioner.getNameFirstRep().getFamily());
 //		}
 	}
+	
+    private void putQuestionnaireResponseValuesIntoETLRecord(ETLRecord etlRecord,
+            QuestionnaireResponse fhirQuestionnaireResponse) {
+        log.trace("putQuestionnaireResponseValuesIntoETLRecord");
+        if (fhirQuestionnaireResponse.hasItem()) {
+            for (QuestionnaireResponseItemComponent item : fhirQuestionnaireResponse.getItem()) {
+                switch (item.getText()) {
```

Comment:
And also , can we check the question by` item.getLinkId()`  since  we used  uuids for the question linkId to ensure uniqueness.
we can just directly copy and paste these  [Fhir constants](https://github.com/I-TECH-UW/Locator-Form/blob/develop/locator-form-webapp/src/main/java/org/itech/locator/form/webapp/fhir/service/FhirConstants.java#L5-L45) across all the projects where this questionareResponse is consumed